### DISCUSSION TITLE
Many to Many Relation

### DISCUSSION THREAD

**Discussion by businessdotkom2020** [Power-Components/livewire-powergrid#407] [Q&A]
Can I Filter Using Many to Many Relations

I have used 
`
            ->addColumn('owners', function (Vehicle $model) {
                $tags = '';
                foreach ($model->owners as $owner) {
                    $tags .= "<span class='tag'>$owner->name</span> ";
                }

                return $tags;
            })
`
to get data but can I filter By Name Here

**Comment by luanfreitasdev** [comment]
Hello. have you tried this?. if it doesn't work I believe you should use join

https://livewire-powergrid.com/#/table/column-filters?id=filter-by-relationship
